test/unit/knex.js

```diff
@@ -414,6 +414,37 @@ describe('knex', () => {
     await trx.executionPromise;
   });
 
+  it('returns false when calling isCompleted on a transaction that is not complete', async () => {
+    const knex = Knex(sqliteConfig);
+    const trxProvider = knex.transactionProvider();
+    const trx = await trxProvider();
+
+    const completed = trx.isCompleted();
+    expect(completed).to.be.false;
+  });
+
+  it('returns true when calling isCompleted on a transaction that is committed', async () => {
+    const knex = Knex(sqliteConfig);
+    const trxProvider = knex.transactionProvider();
+    const trx = await trxProvider();
+
+    await trx.commit();
+
+    const completed = trx.isCompleted();
+    expect(completed).to.be.true;
+  });
+
+  it('returns true when calling isCompleted on a transaction that is rolled back', async () => {
+    const knex = Knex(sqliteConfig);
+    const trxProvider = knex.transactionProvider();
+    const trx = await trxProvider();
+
+    await trx.rollback();
+
+    const completed = trx.isCompleted();
+    expect(completed).to.be.true;
+  });
+
```

Comment:
looks like test is missing where that API is used inside trx handler function.

Like this:

```
knex.transaction(trx => {
   trx.isCompleted();  
});
````

Reply:
Done